buildSrc/src/main/kotlin/io/getstream/chat/android/command/version/codechange/BumpVersion.kt

```diff
@@ -0,0 +1,24 @@
+package io.getstream.chat.android.command.version.codechange
+
+import java.io.File
+
+fun parseVersion(file: File, bumpMinor: Boolean): List<String> {
+    val newLines = file.readLines().map { line ->
+        when {
+            line.trim().startsWith("const val minorVersion =") && bumpMinor -> bumpVersionInLine(line)
+
+            line.trim().startsWith("const val patchVersion =") && !bumpMinor -> bumpVersionInLine(line)
+
+            else -> line
+        }
+    }
+
+    return newLines.toList()
+}
```

Comment:
Could we add a flag here to be notify if, by some reason, `minorVersion` or `patchVersion` are not found?
It will fail if we modify our `Configuration` file in a way that break our logic

Reply:
In case we don't find the version, we won't change the file, because it will hit `else -> line` and the file will stay the same.

Reply:
Nevermind, in case we can't bump the version the user should be aware of it